_posts/2020-08-14-how-to-stay-sane-with-robot-framework.md

```diff
@@ -0,0 +1,252 @@
+---
+layout: post
+title: How to stay sane with robot framework and plaa plaa
```

Comment:
Title is the most important thing for organic visibility, aka search engine optimization, so well to think carefully how to hit common keywords :)